subjects/own-and-return/README.md

```diff
@@ -0,0 +1,42 @@
+## own_and_return
+
+### Instructions
+
+Create a function `only_return` that takes a `struct` Film that has one Field of type `String` and returns the string without owning it.
+
+Create a function `own_and_return` which takes ownership of the struct, and returns the string.
+
+### Expected functions
+
+```rust
+pub struct Film {
+    pub name: String,
+}
+
+pub fn only_return(your_parameter) {
```

Comment:
here usually we put unimplemented code in inline comments, also the return type should be explicit or we should write it must be implemented (here it just seems we do not expect any return).

Actually I think we can say the return is a `String` since there is nothing special about that.

I suggest the follow:
```rust
pub fn only_return(/* to be implemented */) -> String {
```

Reply:
Thanks. Done.

Reply:
I'll resolve this conversation since the pull request is approved